cmd/bblfsh/client.go

```diff
@@ -39,10 +40,19 @@ func (c *clientCmd) Execute(args []string) error {
 		run = c.runStandalone
 	}
 
+	var encoding protocol.Encoding
+	switch(c.Encoding) {
+	case "Base64":
+		encoding = protocol.Base64
+	default:
+		encoding = protocol.UTF8
```

Comment:
Since the parameter is optional, I think a better default would be to leave it undefined